addons/website_sale/views/templates.xml

```diff
@@ -67,14 +67,14 @@
     </template>
 
     <template id="search" name="Search Box">
-        <form t-att-action="keep('/shop'+ ('/category/'+slug(category)) if category else None, search=0)" method="get" t-att-class="_classes">
+        <t t-call="website_sale.website_sale_search_input_group">
+            <t t-set="action" t-value="keep('/shop'+ ('/category/'+slug(category)) if category else None, search=0)"/>
```

Comment:
You set an `action` variable here but you do not use it in the `t-call`ed template...

Reply:
How to do this ?

Reply:
> https://github.com/odoo/odoo/pull/30366#discussion_r259916718

Like you have been doing with all other variables in your task... just use the "action" variable in your template to put it as the value of the `action` attribute of your form (and if `action` is not defined, use '/shop' by default)